easybuild/easyconfigs/t/Theano/Theano-1.0.4-fosscuda-2019b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'Theano'
+version = '1.0.4'
+
```

Comment:
@terjekv Look at the unit test failure regarding `versionsuffix` here.

Reply:
Right, apologies for the delay, I've been otherwise occupied. I'll try to get a look at this sometime this week. Thank you for the pointers!